File: flipkart/parkinglot.cpp
#include<bits/stdc++.h>
using namespace std;

vector < string > split(string s)
{
	vector < string > splitResult;
	string x = "";
	
	for(int i = 0; i < s.size(); i++)
	{
		if(s[i] == ' ')
		{
			splitResult.push_back(x);
			x = "";
		}
		else
			x += s[i];
	}
	splitResult.push_back(x);
	return splitResult;
}

pair<int,int> vacantSpace(vector<vector<int>>grid,int type)
{
	
}

parkVehicle(vector<vecor<int>>&grid,int regNo, string color)
{

}


int main()
{
	string s;
	do
	{
		std::getline(std::cin>>std::ws,s);
		if(s == "exit")
			exit(0);
		vector < string > input = split(s);
		
		if(input[0] == "create_parking_lot")
		{
			int floors = stoi(input[1]), slots = stoi(input[2]); 
			vector < vector < int >>parkGrid(floors,vector<int>(slots,0));
		}
		
		if(input[0] == "parking_vehicle")
		{
			
		}
		else if(input[0] == "unpark_vehicle")
		{
		
		}
		else if(input[0] == "display")
		{
		
		
		} 	
	}while(true);
}

File: flipkart/parkingLot.py
global parkLot
from display import display

def create(id,floor, slot):
	parkGrid = [[0]*int(slot)]*int(floor);
	return parkGrid

def main():
	while(True):
		print("User Input:\n")
		s = input();
		
		if s == "exit":
			exit(0)
		
		inputData = s.split()
		print(inputData)
		if inputData[0] == "create_parking_lot":
			parkLot =  create(inputData[1],inputData[2],inputData[3])
		
		elif inputData[0] == "display":
			display(parkLot, inputData[1],inputData[2])
		
		elif inputData[0] == "park_vehicle":
			park(inputData[1],inputData[2],inputData[3])
			
		elif inputData[0] == "unpark_vehicle":
			park(inputData[1],inputData[2])
		
		showGrid(parkLot)
		

if __name__ == "__main__":
	main()
	
	
	
